pandas/io/json/_normalize.py

```diff
@@ -228,14 +228,17 @@ def json_normalize(
     Returns normalized data with columns prefixed with the given string.
     """
 
-    def _pull_field(js, spec):
+    def _pull_field(js: Dict, spec: Union[List, str]) -> List:
```

Comment:
```js: Dict[str, Any] ```

Reply:
can type parameters be added to the generic List types?